modules/material-management/test/cryptographic_material.test.ts

```diff
@@ -309,6 +352,27 @@ describe('decorateWebCryptoMaterial', () => {
     expect(() => test.setCryptoKey(key2, trace)).to.throw()
   })
 
+  it('Precondition: If the CryptoKey is the only version, the trace information must be set here.', () => {
```

Comment:
It is unclear to me what "If the CryptoKey is the only version" means. Looking at the code, this should be a precondition about the keyring trace during setCryptoKey if the udk isn't already set.

Reply:
If you only set a CryptoKey.  And do not set an unencrypted data key.  This happens in browsers when an EDK can be "unwrapped".  The API lets you do this unwrapping and put the data key into an un exportable CryptoKey.  This means that the unencrypted data key is never available to the javascript.

Reply:
gotcha :+1: